website/docs/d/vra_region_enumeration_aws.html.markdown

```diff
@@ -0,0 +1,28 @@
+---layout: "vra"
+page_title: "VMware vRealize Automation: vra_region_enumeration_aws"
+description: |-
+  Provides a data lookup for vra_region_enumeration_aws.
+---
+
+# Data Source: vra_region_enumeration_aws
+## Example Usages
+
+This is an example of how to read a region enumeration data source for AWS cloud account.
+
+**Region enumeration vSphere data source by its id:**
+```hcl
+data "vra_region_enumeration_aws" "this" {
+  id = vra_region_enumeration_aws.this.id
+}
+```
+## Argument Reference
+
+* `id` - (Optional) The id of the region enumeration for AWS account.
+
+## Attribute Reference
+* `access_key` - Aws Access key ID. Example: ACDC55DB4MFH6ADG75KK
+
+* `secret_key` - Aws Secret Access Key. Example: gfsScK345sGGaVdds222dasdfDDSSasdfdsa34fS
```

Comment:
Both `access_key` and `secret_key` should be moved to Arguments section and mark them as `Required`. Also, remove the examples.